built-in-references/kubernetes/container-allowed-images/constraint.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
```

Comment:
can the "kubernetes" folder be capitalized like "KubernetesService" for consistency?

Reply:
Sure